vesmod/VesEdge/animation.py

```
@@ -0,0 +1,173 @@
+    if len(set(frame_counts)) != 1:
+        raise ValueError(
+            "All animation panels must contain the same number of frames; "
+            f"received {frame_counts}."
+        )
+
+    output_path = Path(path).with_suffix(".gif")
+    fig, axes = plt.subplots(1, len(panels), figsize=figsize, squeeze=False)
+    panel_axes: NDArray[np.object_] = axes[0]
+
+    def animate(frame_index: int) -> None:
+        for panel, ax in zip(panels, panel_axes):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)